index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="styles/index.css">
+  <title>Survey Form</title>
+</head>
+<body>
+  <h1 id="title">Survey Form</h1>
```

Comment:
Stylize a h1 as a tag not necessary to give it an div Id, because it is the best practice to make it easy to change all the h1 at once when changes are needed.

Reply:
- [ ] You can install [Grammerly](https://app.grammarly.com/), which helps scan for common grammatical mistakes,always ensure your comments are clear and easy to understand